packages/devtools_app/lib/src/screens/memory/panes/chart/controller/chart_data.dart

```diff
@@ -16,11 +16,7 @@ enum Json {
   interval,
   isLegendVisible;
 
-  const Json({String? nameOverride}) : _nameOverride = nameOverride;
-
-  final String? _nameOverride;
-
-  String get key => _nameOverride ?? name;
+  String get key => name;
```

Comment:
why not just use `.name` directly? Seems like added complexity to add a getter that just returns the unchanged name value.

Reply:
For me it is alert for future maintainer that key may be different from name, and it is needed for backward compatibility.
But, yes, it is debatable. 
Removed.